Marlin/src/module/tool_change.cpp

```diff
@@ -531,9 +532,74 @@ void tool_change(const uint8_t tmp_extruder, const float fr_mm_s/*=0.0*/, bool n
 
         select_multiplexed_stepper(tmp_extruder);
       #endif
-
+ 
+      // SINGLENOZZLE tool change 
+      #if ENABLED(SINGLENOZZLE)
+      //Load/unload statuses 
+      static bool single_nozzle_load_status[EXTRUDERS]={0};
+      static bool single_nozzle_not_initialised=true;
+      const float old_feedrate_mm_s = feedrate_mm_s;
+	  
+      set_destination_from_current();
+      stepper.synchronize(); 
+      feedrate_mm_s = SINGLENOZZLE_LOAD_FEEDRATE;
+	  
+      //If cold , then rejected and buzzer bip
+      if(!thermalManager.tooColdToExtrude(0)){
```

Comment:
One space after `if` and one space before `{`. See http://marlinfw.org/docs/development/coding_standards.html#brace-style and http://marlinfw.org/docs/development/coding_standards.html#spacing.